src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size } from './theme';
+import { P } from './text';
+import CardList from './card-list';
+import useAllVideos from '../../hooks/use-all-videos';
+import usePodcasts from '../../hooks/use-podcasts';
+
+const Tab = styled('div')`
+    overflow: hidden;
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    margin-left: 38px;
+    margin-right: 38px;
+    margin-top: 50px;
+    margin-bottom: 30px;
+`;
+
+const TabButton = styled('button')`
+    float: right;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    transition: 0.3s;
+    font-size: 18px;
+    font-family: Fira Mono;
+    color: ${colorMap.greyDarkOne};
+    background-color: inherit;
+    border-bottom: 3px solid transparent;
+    width: 136px;
+    padding-top: ${size.micro};
+    padding-bottom: ${size.medium};
+    padding-left: ${size.mediumLarge};
+    padding-right: ${size.mediumLarge};
+    &:hover {
+        color: white;
+    }
+    &:focus {
+        border-bottom-color: ${colorMap.darkGreen};
+    }
+`;
+
+const AllContentStyle = {
+    marginRight: '818px',
+    width: '136px',
+};
+
+export default () => {
+    const { videos, error, isLoading } = useAllVideos();
+    const {
+        podcasts,
+        error: errorPodcasts,
+        isLoading: isLoadingPodcasts,
+    } = usePodcasts();
+
+    const [showVideos, setShowVideos] = useState(false);
+    const [showPodcasts, setShowPodcasts] = useState(false);
+    const [showArticles, setShowArticles] = useState(false);
+    const [showAll, setShowAll] = useState(true);
+
+    const displayVideos = () => {
```

Comment:
I like how we used state hooks to manage which to show! Using four state variables seems a bit high, could we get away with one variable that is something like `'all'` or `'articles'`, etc.? That could remove the need for these four helpers